### PR TITLE
Omnimap-powered maps experience docs updates

### PR DESCRIPTION
**Pull request by akristen**
This PR will be merged sometime on Thurs, Oct 13th. For a TW review, I'd like to request:

- A copy pass for quick fixes before then
- Any notes for fast follows after this has been merged in. If there are urgent notes that should be addressed before it's released, please call that out!

From maps ENG:

- Content is accurate
- Any notes for fast follows on what features we should be prepared to add later.

### CODE REVIEW

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 1]
```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
...
-Some New Relic features, like distributed tracing and incident intelligence, display topological maps that show relationships between your New Relic-monitored entities and other external hosts and services. These maps can help you diagnose and correct issues.
+import omnimapServiceMaps from 'images/omnimapServiceMaps.png'
+
+import omnimapAutomapTooltip from 'images/omnimapAutomapTooltip.png'
+
+import omnimapServiceMapsTopNav from 'images/omnimapServiceMapsTopNav.png'
+
+import omnimapServiceMapsRightPane from 'images/omnimapServiceMapsRightPane.gif'
+
+New Relic can take data from instrumented entities and create topological maps that display the relationships between them. These entity-scoped maps use a source entity as an entry point into the rest of your system, letting you understand how your hosts, services, and applications relate and affect each other.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:21]:
```suggestion
New Relic can take data from instrumented entities and create topological maps that display the relationships between them. These entity-scoped maps use a source entity as an entry point into the rest of your system, letting you understand how your hosts, services, and applications relate to and affect each other.
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 2]
```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
...
+
+If you want to learn more about Workloads maps, then check out [Intro to Workloads](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). If you're looking to learn about automap, then read [Automap for troubleshooting entities](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:23]: decapitalize 'workloads', as we decapitalize other features/capabilities.

Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:23]:
```suggestion
If you want to learn more about Workloads maps, then check out [Intro to Workloads](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). If you're looking to learn about automaps, then read [Automap for troubleshooting entities](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 3]
```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
...
+
+## A consistent maps experience [#omni]
+
+With our omnimap-powered maps experience, we're able to unify disparate map capabilities into a single, consistent experience. Service maps, Workloads maps, and automap share key features and design, like [timewarp](/docs/new-relic-solutions/new-relic-one/ui-data/automaps/#timewarp-cursor), the **Health status** and **Entity types** toggle, and various UI elements.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:27]: this is first mention of 'omnimap' in this doc and I'm not clear what it's referring to. I get that it's 'unifying disparate map capabilities' into a single experience, but I'm not sure what that means from a customer perspective. Is it just a type of map? what does us being able to unify different maps have to do with that single map type? are there different maps under the hood that we're exposing as a single type of map and ,if so, would a customer care about that? Just a bit unclear what the 'omnimap' really is in practical sense.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 4]
```diff
@@ -10,18 +10,42 @@ import contextMapThreeEntities from 'images/context-map-three-entities.png'
...
+
+If you want to learn more about Workloads maps, then check out [Intro to Workloads](/docs/new-relic-solutions/new-relic-one/introduction-new-relic-one). If you're looking to learn about automap, then read [Automap for troubleshooting entities](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
+
+## A consistent maps experience [#omni]
+
+With our omnimap-powered maps experience, we're able to unify disparate map capabilities into a single, consistent experience. Service maps, Workloads maps, and automap share key features and design, like [timewarp](/docs/new-relic-solutions/new-relic-one/ui-data/automaps/#timewarp-cursor), the **Health status** and **Entity types** toggle, and various UI elements. 
+
+On the highest layer, New Relic organizes your maps into three overarching tiers, **UX**, **Services**, and **Infrastructure**. Within these tiers are icons (either circles or hexagons) that represent groups of related entities. Arrows between these groups identify the kind of relationship between groups of entities.
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:29]:
```suggestion
On the highest layer, New Relic organizes your maps into three overarching tiers: **UX**, **Services**, and **Infrastructure**. Within these tiers are icons (either circles or hexagons) that represent groups of related entities. Arrows between these groups identify the kind of relationship between groups of entities.
```

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 5]
```diff
@@ -86,38 +110,49 @@ Here is more detail about the standard visual elements of a map:
...
+Under the timewarp feature, you can explore the far right pane to see an activity stream, your service levels, reported data about metrics, deviations in behavior over time, data about your logs, and view your **Related entities**. Clicking **See full map** in **Related entities** is another method to access automap.
+
+<img 
+  title="Omnimap-powered service maps right pane"
+  alt="A gif that scrolls through the right pane in service maps. It shows different graphs that correspond to the above paragraph."
+  src={omnimapServiceMapsRightPane}
+/>
 
-Distributed trace maps use these colors to indicate health:
+As a note, distributed tracing maps use these colors to indicate health:
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:131]: "as a note" a little hard to parse. Can we just say 'Distributed tracing maps use...' without that?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx [hunk 6]
```diff
@@ -86,38 +110,49 @@ Here is more detail about the standard visual elements of a map:
...
 
 * A solid red hexagon means one or more span errors.
 * A solid orange hexagon means one or more anomalous spans.
 
-## Map tooltip [#tool-tips]
+## Get golden metrics in the maps tooltip [#tool-tips]
 
-Maps in context show additional information about map elements when you hover over them.
+Our omnimap-powered tooltips show a snapshot of an entity's [golden metrics](/docs/apis/nerdgraph/examples/golden-metrics-entities-nerdgraph-api-tutorial). When you select **Map from this entity**, service maps will open up [automap](/docs/new-relic-solutions/new-relic-one/ui-data/automaps).
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:138]: so I might be confused: are there 'automaps' plural? or is there just a single 'automap' concept? I see you using it in singular a few times, so maybe I'm confused how we're meant to be using it.

Comment (zuluecho9) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:138]: another thought maybe for later: it might be good to come up with some style guide entries about our map-related features and what preferred concepts/language is to refer to them.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one.mdx [hunk 7]
```diff
@@ -12,9 +12,9 @@ redirects:
   - /docs/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one
```
Comment (zuluecho9) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one.mdx:12]: a question/reminder about the deleted docs: did you add redirects for both the deleted URLs and the associated redirects of those deleted docs?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx [hunk 8]
```diff
@@ -32,46 +40,63 @@ import notConfiguredHealthStatus from 'images/not-configured-health-status.svg'
...
 <Callout variant="important">
   Automap auto-expand logic relies on the health status of an entity, which is largely determined by the alert status of that entity. For optimal troubleshooting with automap, ensure that your [alerts are properly configured](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/).
 </Callout>
 
 ## Open automap [#open-automap]
 
-Automap appears on any **Related entities** pane for supported entity types:
+To view automap, go to **one.newrelic.com**, then select **APM & services** in the left pane.
+
+1. From the **APM & services** menu, select your source entity.
```
Comment (jchurchmanNR) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:81]: Not sure if it needs to be called out here, but Service Maps can also be found from Mobile Apps, and Browser Apps

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx [hunk 9]
```diff
@@ -201,37 +226,25 @@ The **Map view** button is only available for the following types of entities:
...
 />
 
-Here is an example of how the controls work:
+## Controls: See only what you need to see [#filter-automap]
 
-<Video
-  type="wistia"
-  id="e15c99hsr2"
-/>
+With the **Health status** and **Entity types** dropdowns available to the left of **Timewarp**, you can de-emphasize certain entities on the map by unchecking those options. For example, you can hide the healthy service to focus only on those alerting as you browse the timeline:
```
Comment (jchurchmanNR) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:241]: Health status dropdown only hides the entity health outline around the node(s) -- de-emphasizing them. It does not hide any nodes.

Entity type dropdown _will_ hide entities of the type(s) that are unchecked.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx (hunk 3 above)
Comment (jchurchmanNR) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx:27]: 👍 to @zuluecho9 's point.

Perhaps something along the lines of "With our newest mapping engine, ..."

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx (hunk 8 above)
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:81]: Very useful info and will definitely add that as well, or find a way to work it in.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx (hunk 6 above)
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx]: @akristen how can I tell whether I'm looking at an omnimap-powered map?

The top of this guide seems to say that it's offering an explanation of the omnimap-powered maps, but we discuss other types of maps, such as distributed tracing and external services. That's a bit confusing to me.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/how-use-service-maps.mdx [hunk 10]
```diff
@@ -63,25 +63,17 @@ For best results, update existing agents to the latest version. The required min
...
-
-Each entity in a map displays a color dependent on its performance.
+Each entity displays a color dependent on its performance.
 
 * Green: there are currently no violations for this entities performance.
 * Yellow: there is an open warning violation for this entity.
 * Red: there is an open critical violation for this entity.
-* Gray: no alert conditions have been set for the entity
-* White: agent not reporting. This means that the agent installed on the entity is not reporting any data. This is expected behavior for [databases or externals](#externals).
+* Gray: agent not reporting. This means that the agent installed on the entity is not reporting any data. This is expected behavior for [databases or externals](#externals).
```
Comment (rhetoric101) [inline src/content/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/how-use-service-maps.mdx:75]: @akristen the link to that anchor seems to be broken.

**Review by rhetoric101 — COMMENTED** [review]
@akristen what a herculean task! Great job wrangling these concepts! I inserted a few suggestions and questions.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx [hunk 11]
```diff
@@ -1,12 +1,22 @@
...
+metaDescription: Automap lets you see changes over time of related entities in your system architecture.
 redirects:
   - /docs/new-relic-one/use-new-relic-one/ui-data/automaps
   - /docs/new-relic-one/ui-data/automaps
 ---
 
-import automapsOverview from 'images/automaps-overview.png'
+import omnimapAutomapView from 'images/omnimapAutomapView.png'
+
+import automapOmnimapView from 'images/automapOmnimapView.png'
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:11]: Omnimap == the engine, automap == the capability that uses the engine. Anything that uses omnimap/galaxy will have the same "expression" via UI elements, different capabilities (e.g., timewarp, and everything else covered here). I think you're right that as a fast follow, I'll need to spend more time differentiating what is/is not omnimap as another section of this doc.

[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:11]: I'll ping you about the screenshot!

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx [hunk 12]
```diff
@@ -32,46 +42,63 @@ import notConfiguredHealthStatus from 'images/not-configured-health-status.svg'
...
+To understand entities and relationships, as well as how the entity platform synthesizes both from telemetry data, see [entities](/docs/new-relic-one/use-new-relic-one/core-concepts/what-entity-new-relic/) and [relationships](/docs/apis/nerdgraph/examples/nerdgraph-relationships-api-tutorial/).
 
 <Callout variant="important">
   Automap auto-expand logic relies on the health status of an entity, which is largely determined by the alert status of that entity. For optimal troubleshooting with automap, ensure that your [alerts are properly configured](/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices/).
 </Callout>
 
 ## Open automap [#open-automap]
 
-Automap appears on any **Related entities** pane for supported entity types:
+To view automap, go to **one.newrelic.com**, then select **APM & services** in the left pane.
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx:83]: It's probably easier to say that automap is available in the omnimap tooltip, so you can access automap anytime you see that tooltip prompt. I'll need to confirm @jchurchmanNR, but are there any non-omni maps experiences that automap is still available in?

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx [hunk 13]
```diff
@@ -32,46 +42,63 @@ import notConfiguredHealthStatus from 'images/not-configured-health-status.svg'
...
+  src={automapOmnimapView}
 />
 
 <figcaption>
-  Here's an example of where you could start troubleshooting with automap.
+  Automap shows relationships between entities, letting you navigate around your system architecture to track down issues.
 </figcaption>
 
-## How it works [#how-works]
+You can think of automap as a distinct maps view that breaks a source entity out of groupings shown in the omnimap view via **Service maps** or **Workloads**. In the above screenshot, you can see entities that are two relationships away from your source entity. Compare how automap adjusts to a relationship depth of three instead of two:
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/automaps.mdx]: Went ahead and expanded on this:

> Automap displays relationships between entities in your architecture based on direct calls between services. For example, APM services are represented as vertices (hexagons) and direct calls are represented as edges (lines) on the graph. This shows you how these services depend on one another.
> 
> You can adjust the relationship depth with automap by selecting how many nodes out from your source entity you'd like to see. For example, in the following screenshot, you can see an automap view with the relationship depth set to **Two**:
> 
> [IMAGE OF 2 DEPTH] 
> 
> Compare how automap expands outward from the source entity an additional level when the depth is set to **Three**.
> 
> [IMAGE OF 3 DEPTH]

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx (hunk 6 above)
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx]: Honestly reading the docs back over, I think it's better to default to 'automaps' until told otherwise.

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one.mdx [hunk 14]
```diff
@@ -37,18 +37,18 @@ Here are required versions for APM agents:
...
-  src={newRelicOneDependenciesUi}
+  src={omnimapDependencies}
 />
 
 <figcaption>
-  **[one.newrelic.com](https://one.newrelic.com) > Explorer > (select an entity) > Dependencies**: View a filterable list of all the apps, services, databases, and hosts connected to an entity, and their color-coded health status.
+  **[one.newrelic.com](http://one.newrelic.com) > APM & services > Dependencies**: View a filterable list of all the apps, services, databases, and hosts connected to an entity, and their color-coded health status.
 </figcaption>
 
 You can filter the dependencies page to view specific things that report to the entity. Dependencies include:
```
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/explore-downstream-dependencies-new-relic-one.mdx:54]: Hm, this is an older doc so I'm not sure what the context is. I know that external services are represented in the current maps experience, but the boundaries around this are fairly fuzzy to me too. Might be a good follow up jira to figure this out

File: src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx (hunk 2 above)
[the remark this answers is not in the document]
Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/ui-data/maps-in-context.mdx]: Tbh this was a low view doc that I commandeered for omnimap, but I agree that it'll be very helpful for customers to have a list of these experiences. 

I'm going to add a section at the bottom that links to these other maps, since they're actually documented once in their own docs.

**Review by x8a — DISMISSED** [review]
Amazing work, Alexa!

**Review by zuluecho9 — APPROVED** [review]
